### ISSUE TITLE
Donwload/Upload custom fields not working in Erpnext 9

### ISSUE THREAD

**Issue by lemco1982** [frappe/erpnext#11025]
I've created several simple custom fields in "customer" form in Erpnext 8 and I've used the download function to save them. I've switched to Erpnext 9 and when I tried to upload the custom fields to "customer" form the Erpnext just ignores them and it does nothing...

This happens even if you add a custom field in Eprnext 9 and try to download and upload it. It simply ignores the upload function...

In Erpnext 8 this functionality was working great! Can you please help?

**Comment by lemco1982** [comment]
Any reply?

**Comment by tundebabzy** [comment]
Hi @lemco1982 
Sorry for the delay. Usually we treat stuff like this on the forum and open an issue when it is confirmed that the problem is a bug. That aside, surely there must be some traceback in the browser console. Please verify and let's know it if there is.

**Comment by dpetropoulos-lemco** [comment]
Steps to reproduce problem on a clean v9 installation:
1. Open "Customer" form, select Menu-Customize.
2. Add a row "Test". Update.
3. Refresh browser to make sure the new row shows up.
4. Click "Download". The attached file is the CSV result. The file contains the row "Test".
5. Delete the "Test" row in the browser. Update.
6. Refresh browser to make sure the "Test" row does NOT show up.
7. Click "Upload". Select and upload the just-saved CSV file. Update.
8. The attached image has the browser console error that comes up.
9. The "Test" row is not imported.

![image](https://user-images.githubusercontent.com/27003230/31165730-ccee3704-a8f4-11e7-83ed-833f6907b52f.png)

The attached txt file is actually a CSV. The commenting system does not allow CSV attachments.

[test_row.txt](https://github.com/frappe/erpnext/files/1355014/test_row.txt)

**Comment by lemco1982** [comment]
Any comment?

**Comment by rmehta** [comment]
fixed

**Comment by lemco1982** [comment]
Unfortunately, it is not fixed since a new bug came up! If you click the upload button a pop-up window appears that prompts the user to attach a file. When you click the "Attach" button the new pop-up windows does not give the ability to the user to browse a file or put a link... It seems like hidden or the window is very small... 

Can you please check it?

![screenshot 2017-10-09 12 48 50](https://user-images.githubusercontent.com/9319476/31332754-4d3fab14-acf0-11e7-9a46-0e624855a02b.png)

**Comment by tundebabzy** [comment]
update your bench

**Comment by dpetropoulos-lemco** [comment]
We tested this on a ERPnext 30-day trial cloud, today. It shows up as v9.1.1. How are we executing bench update on a trial cloud? Isn't ERPnext cloud supposed to update automatically?

**Comment by tundebabzy** [comment]
In that case, the changes will soon be live. The fix was released a few minutes age. [(View)](https://github.com/frappe/frappe/releases/tag/v9.1.3)

**Comment by lemco1982** [comment]
Hello again...

We noticed that the cloud has been updated to the following:
Installed Apps
ERPNext: v9.1.2 (master)
Frappe Framework: v9.1.3 (master)

We repeat again the following steps:
1.Open "Customer" form, select Menu-Customize.
2.Add a row "Test". Update.
3.Refresh browser to make sure the new row shows up.
4.Click "Download". The attached file is the CSV result. The file contains the row "Test".
5.Delete the "Test" row in the browser. Update.
6.Refresh browser to make sure the "Test" row does NOT show up.
7.Click "Upload". Select and upload the just-saved CSV file. Update.
8.The attached image has the browser console error that comes up.
9.The "Test" row is not imported.

And the update fuction does not seem to work again...

We get the following error:

![screenshot 2017-10-09 14 36 00](https://user-images.githubusercontent.com/9319476/31336376-4d8df080-acff-11e7-890c-f7319cfafa64.png)
